Formula/kibana.rb

```diff
@@ -45,7 +45,7 @@ def install
     ENV.prepend_path "PATH", prefix/"libexec/node/bin"
     Pathname.new("#{ENV["HOME"]}/.npmrc").write Language::Node.npm_cache_config
     system "npm", "install", "--verbose"
-    system "npm", "run", "build", "--", "--release", "--skip-os-packages", "--skip-archives"
+    system "npm", "run", "build", "--", "--release", "--skip-os-packages", "--skip-archives", "--force"
```

Comment:
I haven't had time to dig into what is going on here, but I don't think we should do this, it's not needed when we build the release bits.